vllm/v1/engine/processor.py

```diff
@@ -96,6 +100,11 @@ def process_inputs(
         sampling_params.update_from_generation_config(
             self.generation_config_fields, eos_token_id)
 
+        # Preprocess multi-modal data
+        mm_inputs = self.mm_input_mapper.process_inputs(
+            decoder_inputs.multi_modal_data, decoder_inputs.mm_processor_kwargs
+        ) if decoder_inputs.multi_modal_data is not None else None
```

Comment:
```suggestion
        ) if not decoder_inputs.multi_modal_data else None
```
I think this is why entrypoint test is failing - `decoder_inputs.multi_modal_data` always returns a dictionary.

Reply:
Thanks good catch!